src/cpp/r/R/Api.R

```diff
@@ -694,10 +668,28 @@
    response$id
 })
 
-.rs.addApiFunction("documentOpen", function(path) {
+.rs.addApiFunction("documentOpen", function(path, 
```

Comment:
Perhaps this should also gain this: 

```r
# validate file argument
   hasFile <- !is.null(filePath) && length(filePath) > 0
   if (hasFile && !is.character(filePath)) {
      stop("filePath must be a character")
   }
   if (hasFile && !file.exists(filePath)) {
      stop(filePath, " does not exist.")
   }
   
   if (hasFile)
   {
      # expand and alias for client
      filePath <- .rs.normalizePath(filePath, winslash = "/", mustWork = TRUE)
      homeDir <- path.expand("~")
      if (identical(substr(filePath, 1, nchar(homeDir)), homeDir)) {
         filePath <- file.path("~", substring(filePath, nchar(homeDir) + 2))
      }
   }
   
```

so that `navigateToFile` would just be an alias.